src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -55,16 +54,72 @@ export default class QuestionnaireForm extends Component {
     }
 
     const items = this.props.qform.item;
-    this.prepopulate(items, newResponse.item, saved_response)
+    this.prepopulate(items, newResponse.item, false)
+    this.state.savedResponse = newResponse;
+  }
+
+  componentDidMount() {
+    this.loadAndMergeForms(this.state.savedResponse);
+  }
+
+  loadPreviousForm() {
+    // search for any QuestionnaireResponses 
+    this.smart.request("QuestionnaireResponse?" + 
+          "&author=" + this.getPractitioner() + 
+          "&subject=" + this.getPatient()).then((result)=>{
```

Comment:
This is run when the user presses the button to pull up previous forms. It will load any form even those that are not in-progress. It was intentional so they could pull up a completed one to start with.